catroid/src/main/java/org/catrobat/catroid/ui/ProjectActivity.java

```diff
@@ -60,6 +60,7 @@
 public class ProjectActivity extends BaseCastActivity implements PlaySceneDialogFragment.PlaySceneInterface {
 
 	public static final String EXTRA_FRAGMENT_POSITION = "FRAGMENT_POSITION";
+	public static final String TAG = ProjectActivity.class.getSimpleName();
```

Comment:
Does this do anything or is it just a left over from logging?

Reply:
Logging leftover.